File: livetester-service/src/main/java/com/yorosis/livetester/vo/EnvironmentVO.java
package com.yorosis.livetester.vo;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class EnvironmentVO {
	private Long environmentId;

	private String environmentName;
	private String targetFolder;
	private String protocol;
	private String host;
	private String port;

	private String logonType;
	private String userName;
	private String password;
	private String pemText;

	private String dbType;
	private String dbHost;
	private String dbPort;
	private String dbName;
	private String dbUsername;
	private String dbPassword;
	private String schemaName;
	private String completionQuery;
	private String tcnQuery;

	private BeneficiaryVO[] beneficiary;
	private ProviderVO[] provider;

}

File: livetester-ui/src/shared/vo/environment-vo.ts
export class EnvironmentVO {
    environmentId: number;
    environmentName: string;
    targetFolder: String;
    protocol: string;
    host: string;
    port: string;
    logonType: string;
    userName: string;
    password: string;
    pemText: string;
    dbType: string;
    dbHost: string;
    dbPort: string;
    dbName: string;
    dbUsername: string;
    dbPassword: string;
    schemaName: string;
    completionQuery: string;
    tcnQuery: string;
}

export class EnvironmentListVO {
    id: number;
    environmentNames: string;
}



